src/components/VItemGroup/VItemGroup.vue

```diff
@@ -70,7 +94,69 @@ export default defineComponent({
     },
   },
   setup(props) {
+    /** @type {import('@nuxtjs/composition-api').Ref<HTMLElement | undefined>} */
+    const nodeRef = ref()
+    const isFocused = ref(false)
     provide(VItemGroupContextKey, props)
+
+    /**
+     * @param {KeyboardEvent} event
+     */
+    const onItemKeyPress = (event) => {
+      if (!arrows.includes(event.key) || !nodeRef.value) return
+
+      event.preventDefault()
+
+      const target = event.target
+
+      // While VItem ultimately renders a button at the moment, that could change in the future, so using a data attribute selector makes it more flexible for the future
+      const items = Array.from(
+        nodeRef.value.querySelectorAll('[data-item-group-item]')
+      )
+
+      const targetIndex = items.findIndex((item) => item === target)
+
+      switch (event.key) {
+        case 'ArrowUp':
+        case 'ArrowLeft':
+          if (targetIndex === 0) {
+            // If navigating up/left on the first item, ignore it
+            return
+          }
+          ensureFocus(items[targetIndex - 1])
+          break
+        case 'ArrowDown':
+        case 'ArrowRight':
+          if (targetIndex === items.length - 1) {
+            // If navigation down/right on the last item, ignore it
+            return
+          }
+          ensureFocus(items[targetIndex + 1])
+          break
+      }
```

Comment:
I think this might need some tweaking for RTL support with the Left/Right handlers.

Reply:
s/could down/right on the last item actually cycle back to the first item?

Reply:
Ah you're right, it should. I missed that in the example.

Reply:
Fixed!